### PR TITLE
Fixes for issue #38, unable to run the plugin on the latest versions …

### PR DESCRIPTION
**Pull request by aflowers001**
…on IntelliJ

Updates were made to ensure compatibility with later versions of Intellij, dropping support for elderly versions along the way.

Also moved the build to the newer Gradle model.

### CODE REVIEW

File: .idea/$CACHE_FILE$ [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="NodePackageJsonFileManager">
+    <packageJsonPaths />
+  </component>
+</project>
```
Comment (szpak) [inline .idea/$CACHE_FILE$]: It's not consistent with .gitignore. Probably you added it first.

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,34 @@
+plugins {
+    id 'java'
+    id 'groovy'
+    id 'org.jetbrains.intellij' version '0.4.16'
+}
+
+group 'com.cholick.idea.spock'
+version '0.12'
```
Comment (szpak) [inline build.gradle]: Wouldn't be better to have a SNAPSHOT version in master - to cover building locally by people?

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#Sat Feb 22 10:28:35 GMT 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.2-all.zip
```
Comment (szpak) [inline gradle/wrapper/gradle-wrapper.properties]: `-bin` version wouldn't be enough?

**Comment by szpak** [comment]
Thanks @aflowers001 for your work! I have no experience with making the Idea plugins, but I put some Gradle-related comments above.

**Comment by computorist** [comment]
Built and installed it and it's running great! Thanks for taking the time to fix it up.

**Comment by aflowers001** [comment]
@szpak Thanks. Just tried to get something working as it is such a useful plugin, and first try with a plugin, let alone one in Gradle.

**Comment by cholick** [comment]
Thank you so much for this PR @aflowers001! Built and successfully installed locally, will get this rolled out to the Jetbrains repo shortly.

**Pull request merged**